### PR TITLE
Add braided river

### PR DESCRIPTION
**Pull request by danroberts728**
Add Braided River Brewing with Untappd provider.

### CODE REVIEW

File: venues/fixtures-al/venues.json [hunk 1]
```diff
@@ -376,6 +376,60 @@
             "tap_list_last_update_time": null
         }
     },
+    {
+        "model": "venues.venue",
+        "pk": 27,
+        "fields": {
```
Comment (drewbrew) [inline venues/fixtures-al/venues.json:382]: I think the merge conflict failed you here. Isn't Chattahoochee already in the file?

Comment (danroberts728) [reply venues/fixtures-al/venues.json:382]: WTH? I did a pull before I pushed the PR. Might be easier for me to figure out on my end.

Comment (danroberts728) [reply venues/fixtures-al/venues.json:382]: Ah. I bet I didn't pull master...

**Comment by danroberts728** [comment]
Going to start this over and try again rather than try to sort this out.

**Pull request closed without merging**